searchcore/src/tests/proton/documentdb/configurer/configurer_test.cpp

```diff
@@ -456,9 +457,13 @@ TEST_F("require that we can reconfigure attribute manager", Fixture)
     ViewPtrs o = f._views.getViewPtrs();
     ReconfigParams params(CCR().setAttributesChanged(true).setSchemaChanged(true));
     // Use new config snapshot == old config snapshot (only relevant for reprocessing)
-    f._configurer->reconfigure(*createConfig(), *createConfig(),
+    auto old_config_snapshot = createConfig();
```

Comment:
I suggest adding a helper function to the Fixture class that handles both prepare_reconfig() and reconfigure() to avoid code duplication in all test functions.

Reply:
Done.